nixos/modules/services/hardware/nvidia-container-toolkit/default.nix

```diff
@@ -66,6 +66,10 @@
 
   config = {
 
+    virtualisation.docker.daemon.settings = lib.mkIf config.hardware.nvidia-container-toolkit.enable {
+      features.cdi = true;
+    };
```

Comment:
Does docker<25 ignore this setting or does it display errors or warnings?

Reply:
Yes, I checked with Docker 24, and it ignores the setting.

Reply:
Good point about warnings: that I didn't realize and didn't explicitly check. I can check tomorrow though.

Reply:
I couldn't see any warnings or anything, but no reason to risk it (we don't know what minor bumps of 24 could change.) I just added the version check too and we can be sure it will be fine.